cli/pkg/earthfile.go

```diff
@@ -24,6 +24,12 @@ type EarthTargets struct {
 	Targets   []string
 }
 
+type Function struct {
```

Comment:
I don't think we need this

Reply:
The current problem that I found is 
```
type Earthfile struct {
	BaseRecipe     spec.Block           `json:"baseRecipe"`
	Path           string               `json:"path,omitempty"`
	SourceLocation *spec.SourceLocation `json:"sourceLocation,omitempty"`
	Targets        []spec.Target        `json:"targets,omitempty"`
	UserCommands   []spec.UserCommand   `json:"userCommands,omitempty"`
	Version        *spec.Version        `json:"version,omitempty"`
}
```
`spec.UserCommand` is deprecated out of the `spec` already. Might have to change to  `Command` in Function -> Block -> Statement  (this I think used to be UserCommand). As we discuss it yesterday.